extide/gradle/src/org/netbeans/modules/gradle/loaders/ModelCache.java

```diff
@@ -100,4 +109,12 @@ private void load() {
     public synchronized State getState() {
         return state;
     }
+
+    private GradleVersion compatibleGradleVersion() {
```

Comment:
Duplicate method - could it be moved to some module-private Util class ?

Reply:
Could find a place for it somewhere...